spacy/lang/sl/lex_attrs.py

```diff
@@ -0,0 +1,116 @@
+    deveto deseto enajsto dvanajsto trinajsto štirnajsto
+    petnajsto šestnajsto sedemnajsto osemnajsto devetnajsto
+    dvajseto trideseto štirideseto petdeseto šestdeseto sedemdeseto
+    osemdeseto devetdeseto stoto tisočo milijonto bilijonto
+    trilijonto kvadrilijonto nešteto
+    
+    prvega drugega tretjega četrtega petega šestega sedmega osmega
+    devega desetega enajstega dvanajstega trinajstega štirnajstega
+    petnajstega šestnajstega sedemnajstega osemnajstega devetnajstega
+    dvajsetega tridesetega štiridesetega petdesetega šestdesetega sedemdesetega
+    osemdesetega devetdesetega stotega tisočega milijontega bilijontega
+    trilijontega kvadrilijontega neštetega
+    
+    prvemu drugemu tretjemu četrtemu petemu šestemu sedmemu osmemu
+    devetemu desetemu enajstemu dvanajstemu trinajstemu štirnajstemu
+    petnajstemu šestnajstemu sedemnajstemu osemnajstemu devetnajstemu
+    dvajsetemu tridesetemu štiridesetemu petdesetemu šestdesetemu sedemdesetemu
+    osemdesetemu devetdesetemu stotemu tisočemu milijontemu bilijontemu
+    trilijontemu kvadrilijontemu neštetemu
+    
+    prvim drugim tretjim četrtim petim šestim sedtim osmim
+    devetim desetim enajstim dvanajstim trinajstim štirnajstim
+    petnajstim šestnajstim sedemnajstim osemnajstim devetnajstim
+    dvajsetim tridesetim štiridesetim petdesetim šestdesetim sedemdesetim
+    osemdesetim devetdesetim stotim tisočim milijontim bilijontim
+    trilijontim kvadrilijontim neštetim
+    
+    prvem drugem tretjem četrtem petem šestem sedem osmem
+    devetem desetem enajstem dvanajstem trinajstem štirnajstem
+    petnajstem šestnajstem sedemnajstem osemnajstem devetnajstem
+    dvajsetem tridesetem štiridesetem petdesetem šestdesetem sedemdesetem
+    osemdesetem devetdesetem stotem tisočem milijontem bilijontem
+    trilijontem kvadrilijontem neštetem
+    """.split()
+)
+
+_currency_words = set(
+    """
+    evro evra evri evre evrov evru evroma evrom evrih evr eur
+    cent centa centi cente centov centu centoma centom centih
+    dolar dolarja dolarji dolarje dolarjev dolarju dolarjema dolarjem dolarjih usd
+    tolar tolarja tolarji tolarje tolarjev tolarju tolarjema tolarjem tolarjih sit
+    dinar dinarja dinarji dinarjev dinarje dinarju dinarjema dinarjem dinarjih din
+    funt funta funti funte funtov funtu funtoma funtom funtih gpb
+    forint forinta forinti forinte forintov forintu forintoma forintom forintih 
+    zlot zlota zloti zlote zlotov zlotu zlotoma zlotom zlotih
+    rupija rupiji rupije rupij rupiju rupijema rupijem rupijih
+    jen jena jeni jene jenov jenu jenoma jenom jenih
+    kuna kuni kune kun kunoma kunam kunah
+    marka marki marke mark markoma markom markih
+    """.split()
+)
+
+
+def like_num(text):
+    if text.startswith(("+", "-", "±", "~")):
+        text = text[1:]
+    text = text.replace(",", "").replace(".", "")
+    if text.isdigit():
+        return True
+    if text.count("/") == 1:
```

Comment:
As far as I can see, this code can never be executed because numbers like `2/3` are split into three parts by the infix rules.

Reply:
I've removed the rule to split numbers on ``

Reply:
The rule still appears to be in the current code?

Reply:
The rule in the punctuation.py or somewhere else? I think I removed it. The infixes are now basically the same as default with the exception of the HYPHENS-PERMITTED rule.

Reply:
Sorry, my mistake, I was looking at the code in `lex_attrs.py` (which obviously doesn't split anything)!